Formula/squashfuse.rb

```diff
@@ -0,0 +1,45 @@
+class Squashfuse < Formula
+  desc "FUSE filesystem to mount squashfs archives"
+  homepage "https://github.com/vasi/squashfuse"
+  url "https://github.com/vasi/squashfuse/archive/0.1.103.tar.gz"
+  sha256 "bba530fe435d8f9195a32c295147677c58b060e2c63d2d4204ed8a6c9621d0dd"
+
+  # Build deps
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
+
+  # Compression algorithms supported by squashfuse
+  depends_on "lz4"
+  depends_on "lzo"
+  depends_on :osxfuse
+  depends_on "squashfs"
+  depends_on "xz"
+  depends_on "zstd"
+
+  def install
+    configure_args = [
+      "--disable-dependency-tracking",
+      "--disable-silent-rules",
+      "--prefix=#{prefix}",
+    ]
+
+    %w[lz4 lz0 xz zlib zstd].each do |algorithm|
+      configure_args << "--with-#{algorithm}=#{HOMEBREW_PREFIX}"
```

Comment:
should use opt prefixes like `Formula["lz4"].opt_prefix` but I doubt the explicit specifications are needed anyway. does it not find the deps on its own?

Reply:
Yeah, just tested it and it was able to find homebrew-installed libs. Will remove this.